packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -66,6 +64,15 @@ const argv = yargs
   .help()
   .argv;
 
+const WATCH_EXPRESSION = [
```

Comment:
Mind moving this back to the top to make the diff more clear and avoid any possible rebase issues?